.travis.yml

```diff
@@ -1,11 +1,42 @@
-language: minimal
+language: generic
+
+matrix:
+  include:
+    - os: linux
+      env:
+        - ARCH=ppc64le
```

Comment:
Miniconda already comes for power.

https://docs.conda.io/en/latest/miniconda.html

I don't think we need to support one

Reply:
ok